client/src/components/index.js

```diff
@@ -1,9 +1,8 @@
 export { default as AccountCard } from './AccountCard';
 export { default as AddUserForm } from './AddUserForm';
 export { default as Banner } from './Banner';
-export { default as Button } from './Button';
+export { default as Action } from './Action';
```

Comment:
should this be a threads Button or Touchable? See earlier comment about the Action component